include/OpenColorIO/OpenColorIO.h

```diff
@@ -1037,23 +1232,45 @@ class OCIOEXPORT ColorSpace
     //!cpp:function:: Clear all the categories.
     void clearCategories();
 
+    ///////////////////////////////////////////////////////////////////////////
+    //!rst::
+    // Encodings
+    // ^^^^^^^^^
+    // It is sometimes useful for applications to group color spaces based on how the color values
+    // are digitally encoded.  For example, images in scene-linear, logarithmic, video, and data
+    // color spaces could have different default views.  Unlike the Family and EqualityGroup
+    // attributes of a color space, the list of Encodings is predefined in the OCIO documentation
+    // (rather than being config-specific) to make it easier for applications to utilize.
+    //
+    // Here is an example config entry that could appear under a ColorSpace:
+    // encoding: scene-linear
+    //
+    // Encoding strings are not case-sensitive. Although users may add their own encodings, the
+    // strings will typically come from a fixed set listed in the documentation (similar to roles).
```

Comment:
We chould define constants for these in `OpenColorTypes.h` like was done for standard roles. Very useful for integration.

Reply:
This can stand as a suggestion for a future PR, since it would not directly effect this work.